### PR TITLE
[CI:DOCS] docs/podman-systemd.unit: Update example to work out of the box + Explicitely mention '.network' units

### PR DESCRIPTION
**Pull request by travier**
docs/podman-systemd.unit: Update example to work out of the box

- Don't order the container unit before local-fs.target as that creates
  an ordering cycle that triggers other issues.
- Use the example network in the container unit
- Only use groups that exists by default for the volume

Signed-off-by: Timothée Ravier <tim@siosm.fr>

---

docs/podman-systemd.unit: Explicitely mention '.network' units

Make the relation between '.container' and '.network' units more
explicit at the beginning.

Signed-off-by: Timothée Ravier <tim@siosm.fr>

---

#### Does this PR introduce a user-facing change?

No, docs only.

```release-note
None
```

### CODE REVIEW

**Comment by travier** [comment]
From working on tests for Fedora CoreOS: https://github.com/coreos/fedora-coreos-config/pull/2216

File: docs/source/markdown/podman-systemd.unit.5.md
Comment (dustymabe) [inline docs/source/markdown/podman-systemd.unit.5.md:32]:
```suggestion
Files with the `.network` extension are only read if they are mentioned in a `.container` file. See the `Network=` key.
```

Comment (travier) [reply docs/source/markdown/podman-systemd.unit.5.md]: Thanks!

**Comment by rhatdan** [comment]
Please hold off on merging this until my PR goes in.

**Comment by travier** [comment]
Link to likely PR to wait for: https://github.com/containers/podman/pull/17351

**Comment by travier** [comment]
Rebased now that https://github.com/containers/podman/pull/17351 has been merged.

File: docs/source/markdown/podman-systemd.unit.5.md
Comment (rhatdan) [inline docs/source/markdown/podman-systemd.unit.5.md:9]: What about kube?

**Comment by rhatdan** [comment]
I think we should add some kind of table to this man page, to make it easier to view. But that would be for another PR.

File: docs/source/markdown/podman-systemd.unit.5.md
Comment (travier) [reply docs/source/markdown/podman-systemd.unit.5.md:9]: Updated

**Comment by rhatdan** [comment]
@travier You need to rebase your PR, since the CI/CD system is broken.  Rebasing will get you the fix.

**Comment by rhatdan** [comment]
LGTM
@containers/podman-maintainers PTAL
Thansk @travier

**Comment by travier** [comment]
Rebased

**Review by vrothberg — APPROVED** [review]
/lgtm
Thank you, @travier !

**Pull request merged**
